src/main/java/com/cloudogu/smeagol/wiki/infrastructure/GitClient.java

```diff
@@ -333,7 +333,7 @@ public FileVisitResult visitFile(java.nio.file.Path file, BasicFileAttributes at
 
 
     private CredentialsProvider credentialsProvider(Account account) {
-        return new UsernamePasswordCredentialsProvider(account.getUsername(), account.getPassword());
+        return new UsernamePasswordCredentialsProvider("__bearer_token", account.getAccessToken());
```

Comment:
I think an AccessTokenProvider would be far more reasonable as the access token itself grant access to the protected resources, even without providing any user information.